src/codecs/resize/resize.ts

```diff
@@ -0,0 +1,46 @@
+import { bitmapToImageData } from '../../lib/util';
+
+type CreateImageBitmapResize = 'pixelated' | 'low' | 'medium' | 'high';
+
+export async function quantize(data: ImageData, opts: ResizeOptions): Promise<ImageData> {
+  let sx = 0;
+  let sy = 0;
+  let sw = data.width;
+  let sh = data.height;
+
+  if (opts.fitMethod === 'cover') {
+    const currentAspect = data.width / data.height;
+    const endAspect = opts.width / opts.height;
+    if (endAspect > currentAspect) {
+      sh = opts.height / (opts.width / data.width);
+      sy = (data.height - sh) / 2;
+    } else {
+      sw = opts.width / (opts.height / data.height);
+      sx = (data.width - sw) / 2;
+    }
+  }
+
+  const bmp = await createImageBitmap(data, sx, sy, sw, sh, {
```

Comment:
Can you use `createImageBitmapPolyfill`? You might have to augment it a bit, but we will have to polyfill it anyway.